File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        d={}
        for i in range(len(nums)+1):
            d[i]=0
        for i in nums:
            d[i]=1
        for i in d:
            if d[i]==0:
                return i

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        int all_sum=0;
        while(n>0){
            all_sum+=n;
            n--;
        }
        for(int i=0;i<nums.length;i++){
            all_sum-=nums[i];
            
        }
        return all_sum;
    }
}